admin/sql/create_tables.sql

```diff
@@ -20,6 +20,7 @@ CREATE TABLE "user" (
   contact_email    CHARACTER VARYING NOT NULL,
   data_usage_desc  TEXT,
   org_name         CHARACTER VARYING,
+  logo_filename    CHARACTER VARYING,
```

Comment:
Then we can remove it afterwards. I think it would be faster to just move all the logos by hand than to write a script. There aren't as many, I'll do it myself.